apis/sgv2-docsapi/src/main/resources/application.yaml

```diff
@@ -108,7 +108,7 @@ quarkus:
           /v2/namespaces/[\\w]+/collections/[\\w]+/batch=/v2/namespaces/{namespace:\\w+}/collections/{collection:\\w+}/batch,
           /v2/namespaces/[\\w]+/collections/[\\w]+/json-schema=/v2/namespaces/{namespace:\\w+}/collections/{collection:\\w+}/json-schema,
           /v2/namespaces/[\\w]+/collections/[\\w]+/upgrade=/v2/namespaces/{namespace:\\w+}/collections/{collection:\\w+}/upgrade,
-          /v2/namespaces/[\\w]+/collections/[\\w]+/(.*?)/.*/function=/v2/namespaces/{namespace:\\w+}/collections/{collection:\\w+}/{document-id}/{document-path:.*}/function,
+          /v2/namespaces/[\\w]+/collections/[\\w]+/(.*?)/?.*/function=/v2/namespaces/{namespace:\\w+}/collections/{collection:\w+}/{document-id: [^/]+}{slash: /?}{document-path: (.+)?}/function,
```

Comment:
This is not valid anymore:

1. new endpoint added
2. other endpoint has changed `.*` to `.+`

Not sure if this is correct, but should be:

```
/v2/namespaces/[\\w]+/collections/[\\w]+/(.*?)/.*/function=/v2/namespaces/{namespace:\\w+}/collections/{collection:\\w+}/{document-id}/{document-path:.+}/function
/v2/namespaces/[\\w]+/collections/[\\w]+/(.*?)/function=/v2/namespaces/{namespace:\\w+}/collections/{collection:\\w+}/{document-id}/function
```